spec/models/food_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  context 'validations' do
+    it 'is not valid without a name' do
+      food = Food.new(name: nil, measurement_unit: 'kg', quantity: 1, price: 0.5)
+      expect(food).to_not be_valid
+      expect(food.errors.full_messages).to include("Name can't be blank")
+    end
+
+    it 'is not valid without a measurement unit' do
+      food = Food.new(name: 'Potato', measurement_unit: nil, quantity: 1, price: 0.5)
+      expect(food).to_not be_valid
+      expect(food.errors.full_messages).to include("Measurement unit can't be blank")
+    end
+
```

Comment:
Please the tests are not running. This error seems to be occurring during the setup of the test fixtures. It appears that there is a not-null constraint on the food_id column of the recipe_foods table, but the test is attempting to insert a null value into this column.

You should check the test fixture data for the recipe_foods table and make sure that all records have a valid food_id value. You may also want to check the test code to ensure that it is properly setting up the fixture data before running the test.

Kindly address these issues.

![image](https://user-images.githubusercontent.com/33840476/229275467-7259e61c-4184-4f92-9511-7bbdb580c2fc.png)